### PR TITLE
Update compliance docs

### PR DESCRIPTION
**Pull request by wjwoodson**
* Move federal requirements and recommendations from `custom domains` to general compliance
* Add notes for DNSSEC support in CDN broker
* General updates in custom domains and CDN broker docs

### CODE REVIEW

File: content/docs/services/cdn-route.md [hunk 1]
```diff
@@ -209,6 +213,12 @@ Other headers, such as HTTP auth, are stripped by default.
 
 If you need a different configuration, contact [cloud.gov support](/help/).
 
+### DNSSEC Support
+
+Custom domains using the CDN broker will not fully validate DNSSEC (between your CNAME record and cloudfront.net). This is because the AWS CloudFront service does not currently support DNSSEC. For a complete implementation of DNSSEC, we recommend instead running a proxy server within your boundary that forwards to your application in cloud.gov, or using an alternative CDN that supports DNSSEC.
```
Comment (wjwoodson) [inline content/docs/services/cdn-route.md]: Drafted some recommendations for alternate implementations in `custom-domains.md` but these are probably out of scope and would take more work to be useful to potential users.

## Additional custom domain options

While the preferred way to set up custom domains is through the cloud.gov [CDN service]({{< relref "docs/services/cdn-route.md" >}}), you can set up a custom domain without it. You'll need a proxy server within your boundary or your own CDN service which terminates TLS connections:

### Proxy server

1. Obtain a TLS certificate for your preferred domain, and configure your proxy to present that certificate.
1. Point your domain to the proxy server via `A` and `AAAA` DNS records
1. Configure the proxy `ProxyPass` to forward traffic to your app `APPNAME.app.fr.cloud.gov`

### Custom CDN service

1. Obtain a TLS certificate for your preferred domain, and configure your CDN to present that certificate.
1. Point your domain to the CDN service via `CNAME` DNS record
1. Configure the CDN service `Origin` to forward traffic to your app `APPNAME.app.fr.cloud.gov`

 
***Note: need to work out general behavior for Host: header. CF Router needs to know this is for the corresponding customer app. ***

**Review by pburkholder — DISMISSED** [review]
These changes really help organize these details. 

I think keeping the DNSSEC alternatives as they are, with just a high-level reference, is better then getting into the details of what in implementation might look like.

File: content/docs/compliance/federal-requirements-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: Federal Requirements and Recommendations
+---
+
+## IPv6
+
+cloud.gov ensures all applications are accessible over IPv6. You don't have to take any action.
+
+## HTTPS
+
+cloud.gov ensures all applications are accessible only over HTTPS with [HTTP Strict Transport Security (HSTS) headers](https://https.cio.gov/hsts/) in accordance with the [HTTPS-Only Standard](https://https.cio.gov/). You don't have to take any action.
```
Comment (jmcarp) [inline content/docs/compliance/federal-requirements-recommendations.md]: Is it worth documenting the 301 from HTTP to HTTPS here also?

Comment (brittag) [reply content/docs/compliance/federal-requirements-recommendations.md]: Yes!

File: content/docs/compliance/federal-requirements-recommendations.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+menu:
+  docs:
+    parent: compliance
+title: Federal Requirements and Recommendations
```
Comment (brittag) [inline content/docs/compliance/federal-requirements-recommendations.md]: Is there a slightly more specific title we can use? "Federal requirements for domains" ?

File: content/docs/services/cdn-route.md (hunk 1 above)
Comment (brittag) [reply content/docs/services/cdn-route.md]: That makes sense to me. Probably just put in a note that they can contact us if they need implementation recommendations? And we can send them these notes and work out the details with them.

**Comment by brittag** [comment]
I made a few changes to try to make this more user-centered:

* On custom domains page: link to compliance page.
* On compliance page: rename title from "Federal Requirements and Recommendations" to "IPv6, HTTPS, and DNSSEC", to help people find the info when skimming for keywords.
* On compliance page: add intro that links to custom domains page.
* On compliance page: cross-reference more SSP controls.
* On compliance page: add link to CDN service page DNSSEC implementation advice.
* On CDN service page: consolidate before-you-start DNSSEC guidance into main DNSSEC section, and link to that section before you start.
* On CDN service page: add a note to DNSSEC implementation advice that readers should contact us for help.

I think these docs may still be somewhat confusing to readers, but I believe at least they will be better than before.

**Review by mogul — APPROVED** [review]

**Review by mogul — APPROVED** [review]

**Pull request merged**